CMakeLists.txt

```diff
@@ -27,7 +27,17 @@ include(cmake/CPM.cmake)
 CPMAddPackage(
   NAME PackageProject.cmake
   GITHUB_REPOSITORY TheLartians/PackageProject.cmake
-  VERSION 1.4
+  VERSION 1.4.1
+)
+
+# Note: If fmt is not imported, this is needed to prevent: CMake Error: install(EXPORT
+# "GreeterTargets" ...) includes target "Greeter" which requires target "fmt" that is not in any
+# export set.  see too https://gitlab.kitware.com/cmake/cmake/-/issues/15415
+CPMAddPackage(
+  NAME fmt
+  GIT_TAG 7.1.3
+  GITHUB_REPOSITORY fmtlib/fmt # to get an installable target
+  OPTIONS "FMT_INSTALL YES"
```

Comment:
I don't think the comments are required as `FMT_INSTALL` (and the install errors when it's removed) should be self-explanatory.

Reply:
ok